packages/main/src/components/admin-sidebar/AdminSidebar.jsx

```diff
@@ -0,0 +1,311 @@
+            Customize
+          </Link>
+          <Link
+            onClick={() => setModal(!openModal)}
+            className={styles.sidebarLink}
+            to="/admin/settings/aboutworkspace"
+          >
+            <FiAlertOctagon className={styles.icon} />
+            About this workspace
+          </Link>
+        </div>
+      </div>
+      <div className={styles.sidebarMenu}>
+        <h6 className={styles.sidebarHeading}>ADMINISTRATION</h6>
+        <div className={styles.sidebarMenuItems}>
+          <NavLink
+            exact
+            onClick={() => setModal(!openModal)}
+            activeClassName={styles.sidebarLinkActive}
+            className={styles.sidebarLink}
+            to="/admin/settings"
+          >
+            <FiSettings className={styles.icon} />
+            Settings & permission
+          </NavLink>
+          <Link
+            onClick={() => setModal(!openModal)}
+            className={styles.sidebarLink}
+            to="/admin/settings/managemembers "
+          >
+            <FiBook className={styles.icon} />
+            Manage members
+          </Link>
+          <Link
+            onClick={() => setModal(!openModal)}
+            className={styles.sidebarLink}
+            to="/admin/settings/usergroups"
+          >
+            <FiUsers className={styles.icon} />
+            User groups
+          </Link>
+          <Link
+            onClick={() => setModal(!openModal)}
+            className={styles.sidebarLink}
+            to="/admin/invites"
+          >
+            <FiUserPlus className={styles.icon} />
+            Invitations
+          </Link>
+          <NavLink
+            exact
+            onClick={() => setModal(!openModal)}
+            activeClassName={styles.sidebarLinkActive}
+            className={styles.sidebarLink}
+            to="/admin/settings/billings"
+          >
+            <FiCreditCard className={styles.icon} />
+            Billings
+          </NavLink>
+          <Link
+            onClick={() => setModal(!openModal)}
+            className={styles.sidebarLink}
+            to="/"
+          >
+            <FiScissors className={styles.icon} />
+            Authentication
+          </Link>
+          <Link
+            onClick={() => setModal(!openModal)}
+            className={styles.sidebarLink}
+            to="/admin/settings/deprecation"
+          >
+            <FiScissors className={styles.icon} />
+            Deprecation
+          </Link>
+          <Link
+            onClick={() => setModal(!openModal)}
+            className={styles.sidebarLink}
+            to="/"
+          >
+            <FiCheckSquare className={styles.icon} />
+            About this workspace
+          </Link>
+          <Link
+            onClick={() => setModal(!openModal)}
+            className={styles.sidebarLink}
+            to="/"
+          >
+            <FiAlertCircle className={styles.icon} />
+            Support for Transport layer security
```

Comment:
some of these links aren't necessary

Reply:
Restructuring the page by removing irrelevant links is on another linear tickets....we are trying to reduce lot of changes per PR